azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/launcher/CallingCompositeJavaLauncher.java

```diff
@@ -34,11 +36,21 @@ public void launch(final CallLauncherActivity callLauncherActivity,
                        final String meetingLink,
                        final Function1<? super String, Unit> showAlert) {
 
-        final CallComposite callComposite =
-                new CallCompositeBuilder()
-//                        .theme(new ThemeConfiguration(R.style.MyCompany_Theme))
-                        .build();
 
+        final CallCompositeBuilder builder = new CallCompositeBuilder();
```

Comment:
Is branch up to date? It's missing
        final CallCompositeBuilder builder = new CallCompositeBuilder();

        if (AdditionalFeatures.Companion.getSecondaryThemeFeature().getActive()) {
            builder.theme(new ThemeConfiguration(R.style.MyCompany_Theme_Calling));
        }


For secondary theme.

Reply:
Branch is uptodate with feature/localization. Kotlin Launcher consists of theme implementation. Java launcher didn't have this. Adding this now. Thanks :)

Reply:
Hmmm, strange it's missing, it's in my branch, either way thanks.